### PR TITLE
Firmware, kernel, npm, Isar, kas updates

### PR DESCRIPTION
**Pull request by jan-kiszka**
Specifically allows us to move forward with upstreaming U-Boot as it acts as in integration reference and source for binary-only artifacts.

### CODE REVIEW

File: conf/distro/debian-buster-backports.list [hunk 1]
```diff
@@ -0,0 +1,2 @@
+deb	http://deb.debian.org/debian buster-backports main
+deb-src	http://deb.debian.org/debian buster-backports main
```
Comment (BaochengSu) [inline conf/distro/debian-buster-backports.list:2]: should the backports be added into snapshot list?

Comment (jan-kiszka) [reply conf/distro/debian-buster-backports.list:2]: I can add it there as placeholder - reproducing the new npm version from the current snapshot date is not working, obviously. But it should be updated on next freeze, yet.

Comment (BaochengSu) [reply conf/distro/debian-buster-backports.list:2]: It's OK, we can schedule the modification by next release, currently just a placeholder or even a todo comment is sufficient.

Comment (jan-kiszka) [reply conf/distro/debian-buster-backports.list:2]: Done in the updated version.

**Pull request merged**